### PR TITLE
GDAL with OGDI

### PR DESCRIPTION
**Pull request by hummeltech**
@mattjdnv needs OGDI support for GDAL

### CODE REVIEW

File: SPECS/ogdi.spec [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+License:        BSD
+URL:            http://ogdi.sourceforge.net/
+Source0:        https://github.com/libogdi/ogdi/archive/%{name}_%{tag}.tar.gz
+Source1:        http://ogdi.sourceforge.net/ogdi.pdf
+
+BuildRequires:  expat-devel
+BuildRequires:  gcc
+BuildRequires:  libtirpc-devel
+BuildRequires:  make
+BuildRequires:  tcl-devel
```
Comment (jbronn) [inline SPECS/ogdi.spec]: Ditch TCL.

File: SPECS/ogdi.spec [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+%package odbc
+Summary:	ODBC driver for OGDI
+Requires:	%{name} = %{version}-%{release}
+
+%description odbc
+ODBC driver for OGDI.
+
+
+%package tcl
```
Comment (jbronn) [inline SPECS/ogdi.spec]: Ditch TCL.

File: SPECS/ogdi.spec [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+%configure \
+	--with-binconfigs \
+	--with-expat \
+	--with-zlib
+
+# WARNING !!!
+# using %{?_smp_mflags} may break build
+%{__make}
+
+# build tcl interface
```
Comment (jbronn) [inline SPECS/ogdi.spec]: Ditch TCL.

File: SPECS/ogdi.spec [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+%{_bindir}/%{name}-config-%{cpuarch}
+%{_libdir}/pkgconfig/%{name}.pc
+%dir %{_includedir}/%{name}
+%{_includedir}/%{name}/*.h
+%{_libdir}/libogdi.so
+
+%files odbc
+%{_libdir}/%{name}/liblodbc.so
+
+%files tcl
```
Comment (jbronn) [inline SPECS/ogdi.spec]: Ditch TCL.

**Review by mattjdnv — APPROVED** [review]
Outstanding!

**Review by jbronn — CHANGES_REQUESTED** [review]
I did this recently for EL9 because I recently noticed references to the format in Hoot's source.  I didn't backport to EL7 because no one had complained about it in 4 years -- I originally ditched OGDI in `hoot-gdal`, the predecessor.  So please incorporate changes from EL9's:
* [ogdi.spec](https://github.com/radiant-maxar/geoint-deps/blob/stable-el9/SPECS/ogdi.spec): specifically get rid of TCL-related stuff we'll never use and add this patch:
* [ogdi-4.1.0-sailer.patch](https://github.com/radiant-maxar/geoint-deps/blob/stable-el9/SOURCES/ogdi-4.1.0-sailer.patch): An issue fixed by [RedHat](https://bugzilla.redhat.com/show_bug.cgi?id=1470896) from their latest RPM code (`rawhide` branch IIRC).

**Review by jbronn — APPROVED** [review]
LGTM

**Review by BuechnerGIS — APPROVED** [review]
:rocket:

**Pull request merged**